config/locales/fr.yml

```diff
@@ -56,9 +56,9 @@ fr:
       title: "Fournir un code COVID Shield au patient"
       subheading: "Demandez au patient de :"
       step1: "Appuyer sur l'icône pour ouvrir l'application COVID Shield."
-      step2: "Appuyer sur la section blanche au bas de l'écran."
+      step2_html: "Appuyer sur la section blanche au bas de l'écran."
```

Comment:
To map English new accessibility string (`Tap the COVID Shield status section button at the bottom of the screen.`), 

I suggest:
`Appuyer sur le bouton de la section du statut de COVID Shield au bas de l'écran`

Not sure how it should be split with the strong and hidden tags since word order is changed.